src/pages/survey/create.page.tsx

```diff
@@ -1,18 +1,31 @@
+import { useRouter } from 'next/router';
 import { css } from '@emotion/react';
 
 import CTAButton from '~/components/button/CTAButton';
 import Header from '~/components/header/Header';
+import { SURVEY_PATH } from '~/constants/path';
+import CreateStopDialog from '~/features/survey/addSurveyForm/CreateStopDialog';
 import CreateSurvey from '~/features/survey/CreateSurvey';
 import { fixedBottomCss } from '~/features/survey/styles';
+import useBoolean from '~/hooks/common/useBoolean';
 
 const CreateSurveyPage = () => {
+  const router = useRouter();
```

Comment:
```suggestion
  const router = useInternalRouter();
```

사실 이런 훅을 만들어두긴 했습니다요 ...!